sdk/storage/ci.webjobs.yml

```diff
@@ -0,0 +1,37 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger: none
```

Comment:
We should consider naming this `ci.functions.yml` to keep consistent with the other functions-specific pipelines.

Reply:
Fixed.